src/main/java/com/apuope/apuope_re/controller/testController.java

```diff
@@ -0,0 +1,19 @@
+package com.apuope.apuope_re.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api")
+public class testController {
+    @GetMapping("/test")
+    public String test() {
+        return "Test...";
+    }
+
+    @GetMapping("/test2")
+    public String test2() {
+        return "Test2...";
+    }
+}
```

Comment:
I guess you have used this for testing the JWT, but it should not be included to merge.

Reply:
yes I agreed, I just left them for testing purposes